packages/gatsby-source-wordpress/src/steps/source-nodes/index.ts

```diff
@@ -8,9 +8,14 @@ import store from "~/store"
 import fetchAndCreateNonNodeRootFields from "./create-nodes/fetch-and-create-non-node-root-fields"
 import { allowFileDownloaderProgressBarToClear } from "./create-nodes/create-remote-file-node/progress-bar-promise"
 import { sourcePreviews } from "~/steps/preview"
+import { hasStatefulSourceNodes } from "~/utils/gatsby-features"
 
 const sourceNodes: Step = async helpers => {
-  const { cache, webhookBody, refetchAll } = helpers
+  const { cache, webhookBody, refetchAll, actions } = helpers
+
+  if (hasStatefulSourceNodes) {
+    actions.enableStatefulSourceNodes()
```

Comment:
```
packages/gatsby-source-wordpress/src/steps/source-nodes/index.ts:17:13 - error TS2339: Property 'enableStatefulSourceNodes' does not exist on type 'Actions'.

17     actions.enableStatefulSourceNodes()
               ~~~~~~~~~~~~~~~~~~~~~~~~~


Found 1 error in packages/gatsby-source-wordpress/src/steps/source-nodes/index.ts:17
```

Seems like we are also missing some types for this in https://github.com/gatsbyjs/gatsby/blob/ccb0e7a153e0866f3991c25c9d1a27e345c625d1/packages/gatsby/index.d.ts#L1291

It seems like there this action was added wrongly to `GatsbyNode` API list and not `Actions`:

https://github.com/gatsbyjs/gatsby/blob/ccb0e7a153e0866f3991c25c9d1a27e345c625d1/packages/gatsby/index.d.ts#L437-L440

Reply:
This was resolved in https://github.com/gatsbyjs/gatsby/pull/37933 but this PR need to be rebased or master merged in

Reply:
merged master in https://github.com/gatsbyjs/gatsby/pull/37920/commits/d2ea933ddc9801c783962c9b3e7e5095ca096283 so this is resolved now